src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -24,6 +29,35 @@
 PIM_FLUENT_PRODUCT_VERSION = FLUENT_VERSION.replace(".", "")
 
 
+class LaunchModes(Enum):
+    """Contains the standard fluent launch modes."""
+
+    MESHING_MODE = [Meshing, True]
+    PURE_MESHING_MODE = [PureMeshing, True]
+    SOLVER = [Solver, False]
+    SOLVER_LITE = [SolverLite, False]
+
+    @staticmethod
+    def get_mode(mode: str) -> "LaunchModes":
+        """Returns the LaunchMode based on the mode in string format."""
+        try:
+            mode = MODE_DICT[mode]
+        except KeyError:
+            LOG.info("Please provide an option amongst the 4 allowed modes.")
+            for i, mode in enumerate(MODE_DICT.keys()):
+                LOG.info(i + 1, ". ", mode)
+            raise RuntimeError("The passed mode matches none of the 4 allowed modes.")
+        return mode
+
+
+MODE_DICT = {
```

Comment:
This information can be added in the LaunchModes class to its each member. The enum class can be iterated over its members to find out the match.

Reply:
Thanks for the suggestion. This has been implemented as discussed.